devtools/conda-envs/psi-nightly.yaml

```diff
@@ -13,7 +13,7 @@ dependencies:
   - pyyaml
   - py-cpuinfo
   - psutil
-  - qcelemental >=0.4.2
+  - qcelemental >=0.6.0
```

Comment:
want to add msgpack to this lane and leave plain psi to json to get some testing variety?